charts/common/templates/_extraresources.tpl

```diff
@@ -0,0 +1,9 @@
+{{- define "accelleran.common.extraResources" -}}
+{{- $ := get . "top" | required "The top context needs to be provided to common drax license enabled" -}}
+{{- $values := get . "values" | default $.Values -}}
+
+{{- range $values.extraResources }}
```

Comment:
Do we need to add a check whether the extraResources exist?

Reply:
We maybe can add a check to see if it is a list specifically to give a bit of a clearer error message. If it's not helm will complain anyway as it wouldn't be able to iterate over it...

Reply:
The check became nil or kind list (ie slice as it's determined by reflection in go) being fine.

Nil is a special case that is also the default if `extraResources` is not set explicitly set to an empty list in a chart's values. `range` handles it like it would an empty list, which is the desired default behaviour.